quarkus-extension/engine/deployment/src/test/java/org/camunda/bpm/quarkus/engine/test/config/CamundaEngineProgrammaticAndConfigFileTest.java

```diff
@@ -71,7 +71,7 @@ public void shouldUseProvidedConfigurationAndConfigProperties() throws SQLExcept
     // assert engine config properties
     assertThat(configuration.isCmmnEnabled()).isFalse();
     assertThat(configuration.isDmnEnabled()).isFalse();
-    assertThat(configuration.getHistory()).isEqualTo("none");
+    assertThat(configuration.getHistory()).isEqualTo("full");
```

Comment:
This should be `none` since we set it in the `mixed-application.properties` file which has a higher priority.